lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -1337,13 +1337,14 @@ def cmd_load(*args)
         if (client.core.use(modulenameprovided) == true)
           add_extension_client(md)
 
-          if md == 'stdapi' && !client.exploit_datastore['AutoLoadStdapi'] && client.exploit_datastore['AutoSystemInfo']
+          if md == 'stdapi' && (client.exploit_datastore && !client.exploit_datastore['AutoLoadStdapi'] && client.exploit_datastore['AutoSystemInfo'])
```

Comment:
This was previously crashing as `client.exploit_datastore` was nil. I believe there's missing functionality here, but that's not a blocker for now